rust/processor/src/processors/default_processor.rs

```diff
@@ -152,6 +156,36 @@ fn insert_transactions_query(
     )
 }
 
+#[derive(Clone, Debug)]
+pub struct TransactionModelBatch {
+    start_version: StartVersion,
+    end_version: EndVersion,
+    items_to_insert: Vec<TransactionModel>,
```

Comment:
can these be `&[TransactionModel]`?

Reply:
i asked also in your PR. when do we use `&[TransactionModel]` vs `&Vec<TransactionModel>`?